contributors/design-proposals/bootstrap-discovery.md

```diff
@@ -101,7 +101,7 @@ An interesting aspect here is that this information is often easily obtained bef
 The user experience for joining a cluster would be something like:
 
 ```
-kubeadm join --token=ae23dc.faddc87f5a5ab458 <address>
+kubeadm join --token=ae23dc.faddc87f5a5ab458 <address>:<port>
```

Comment:
is port optional? is address a fully formed URL (which will imply a port) or a raw IP address / DNS name?